packages/web-components/src/components/va-featured-content/va-featured-content.tsx

```diff
@@ -16,8 +16,19 @@ export class VaFeaturedContent {
    */
   @Prop() uswds?: boolean = false;
 
+  /**
+   * Local state for slot=headline's text.
+   * Used to place an aria-label for role="region" with the same text as the heading.
+   */
+  @State() headlineText: string = null;
```

Comment:
I wonder if `@State` is necessary? Would this value only be assigned once when the component loads? Does this still work if you just assign the value to a private variable?

Reply:
Good call, I updated that to a private variable.